apps/frontend/src/pages/projects/components/project-create-form/styles.module.css

```diff
@@ -5,6 +5,16 @@
 	align-items: flex-end;
 }
 
+.description-counter {
+	position: relative;
+	bottom: 8px;
```

Comment:
I don't quite understand why we need this in bottom: 8px or negative margin in the layout? shouldn't it just be at the same place where error message is usually shown?

Reply:
If I remove it, there will be 16 px between the counter and the block, and I need 8 px (as well as between the error message and the block)

Reply:
Why there will be 16 px gap? is this because of flex gap on form container? if so, it's better to wrap input and counter into wrapper to group them